Readme.md

```diff
@@ -3,6 +3,50 @@ TOML Sort
 
 Simple tool for sorting toml files via cli or ci. It was made with `Cargo.toml` files in mind but is likely useful elsewhere.
 
+## Usage
+
+### Installation
+
+```sh
+cargo install --git https://github.com/Off-Narrative-Labs/toml_sort
+```
+
+### Sorting files
+
+```sh
+toml-sort [files]
+```
+
+### Checking sorted files
+
+```sh
+toml-sort --check [files]
+```
+
+### GitHub Action
+
+This will only check `./Cargo.toml`:  
+```yaml
+- uses: Off-Narrative-Labs/toml_sort
+```
+
+If you want the action to check all the TOML files, then you can use:
+
+```yaml
+- uses: Off-Narrative-Labs/toml_sort
+  with:
+    all: true
+```
+
+In alternative, if you want to check only a certain set of files:
+
+```yaml
+- uses: Off-Narrative-Labs/toml_sort
+  with:
+    files: Cargo.toml foo/Cargo.toml bar/Cargo.toml
```

Comment:
yaml supports lists right? Should we use a list here instead of parsing based on spaces? (Not a blocker though)

Reply:
since the tool automatically parses a list in a single string, divided by spaces, it wouldn't make any sense to pass a list that would be converted to this format

Reply:
Oh, I see. `toml-sort` itself wants a space delimited list. Makes sense to me.